geomesa-nifi-processors/src/main/scala/org/locationtech/geomesa/nifi/GeoMesaIngestProcessor.scala

```diff
@@ -59,24 +60,41 @@ class GeoMesaIngestProcessor extends AbstractProcessor {
   @volatile
   private var converter: convert.SimpleFeatureConverter[_] = null
 
-
   @OnScheduled
   def initialize(context: ProcessContext): Unit = {
-    dataStore = getDataStore(context)
-    val sft = getSft(context)
-    dataStore.createSchema(sft)
 
-    converter = getConverter(sft, context)
-    featureWriter = createFeatureWriter(sft, context)
-    getLogger.info(s"Initialized GeoMesaIngestProcessor datastore, fw, converter for type ${sft.getTypeName}")
+    try {    
+      val zookeepers = context.getProperty(Zookeepers).getValue
```

Comment:
i think maybe this should be its own function called checkZookeepers() or something